documentation/general/dash-sonic-hld.md

```diff
@@ -1527,3 +1575,61 @@ For the example configuration above, the following is a brief explanation of loo
 		h. Third Action is Appliance Encap for id 22
 		i. Packet shall be encapsulated with Outer DIP as 100.8.1.2 and SIP as VIP of this originating appliance card with VNI of 101. 
 		j. Inbound flow shall be similar to PL and outer encap shall be of the SLB MUX and not of the NSG appliance.
+
+# 4 SONiC DASH virtual switch
+## 4.1 Motivation
+
+1. Provide a Proof of Concept (POC) for development and collaboration. Utilizing the typical SONiC workflow, we can leverage this virtual switch image to construct a virtual testbed, eliminating the need for a complete hardware device. This virtual DPU image enables the creation of a mixed hardware-software testbed or a software-only testbed, applicable to both the control plane and the data plane.
+2. Enable Continuous Integration(CI) via Azure Pipelines (Azp) for SONiC repositories, like sonic-buildimage, sonic-swss and so on.
+
+## 4.2 Architecture
+
+![BMv2 virtual SONiC](images/bmv2-virtual-sonic.svg)
+
+## 4.3 Modules
+
+### 4.3.1 BMv2 (dataplane engine)
+
+This component is the original P4 BMv2 container image, which serves as the implementation of the dataplane.
+It attaches three types of interfaces: Ethernet, eth, and cpu.
```

Comment:
maybe better to give the "cpu" a better name.

Reply:
i think we can probably give all these ports better names... the type doesn't have to match with the physical port name. it should give people a better understanding of the architecture when reading the doc.

Reply:
Done